modules/acs-architecture-external-components.adoc

```diff
@@ -6,7 +6,10 @@
 = External components
 In addition to the primary services, {product-title} also interacts with other entities to provide enhanced security for your clusters.
 
-Figure 3 depicts the {product-title-short} architecture for {ocp}. In other Kubernetes based environments, Scanner services are only installed once with the centralized  service.
+Figure 3 depicts the {product-title-short} architecture for {ocp}; note the following exceptions:
```

Comment:
I would revise this a little:

s/Figure 3 depicts the {product-title-short} architecture for {ocp}; note the following exceptions:/Figure 3 depicts the {product-title-short} architecture for {ocp}, but you should note the following exceptions: